parser.ts

```diff
@@ -116,6 +120,7 @@ export class Parser {
     this.isLine = false;
     this.lineNo = null;
     this.line = ws1 + this.current.text;
+    this.lineSource = null;
```

Comment:
We can't set this to `(ws1, this.current.text, '', '')` because we don't know yet if `this.current.text` represents a lineNo or not. We *could* potentially store this as `(ws1, '', '', this.current.text)`, with the assumption that we "rewrite" it when we encounter a line number.